apps/content_filters/filter_condition/filter_condition_field.py

```diff
@@ -242,3 +245,22 @@ def is_in_article(self, article):
 
     def get_value(self, article):
         return article['extra'].get(self.field.name, '')
+
+
+class FilterConditionFeatueMediaField(FilterConditionField):
+    def __init__(self, field):
+        self.field = FilterConditionFieldsEnum.featuremedia
+        self.entity_name = 'associations.featuremedia._id'
+        self.field_type = bool
+
+    def get_value(self, article):
+        return article.get('associations', {}).get('featuremedia', {}).get('_id')
```

Comment:
I think `featuremedia` can be `None` (when you remove association) and this would raise exception

Reply:
fixed